### ISSUE TITLE
Update MPlayer (for Windows)

### ISSUE THREAD

**Issue by wladimirleite** [sepinf-inc/IPED#2056]
We are using a relatively old version of MPlayer (from 2018) for Windows. 
Although changes are minimal, I saw some improvements in VideoThumbTask using a newer version (e.g. some videos that older version were unable to skip frames, now are handled fine).
I will run a larger test to check performance and generated thumbs quality, and see if it is safe to update.

**Comment by wladimirleite** [comment]
Unfortunately the most recent version available, from December/2022 (no update in the last year!), doesn't output the "rotation" information.
That affects a lot of videos, making the "quality" of generated thumbs worse.
As performance is basically the same as the previous version and the "quality" benefit is limited to a small subset of the videos I tested, it is not worth updating now.

I will check again in the future, and maybe try to compile from the source, but for now I will close this.

**Comment by wladimirleite** [comment]
Recently, #2863 was implemented as a fallback to process videos not handled by MPlayer (reported in #2537).
@gmdalpian pointed out that the most recent MPlayer for Windows build does support these videos (or at least some of them).
I tested the samples I have here and confirmed that the newer version has better support for newer codecs.
I will run more tests to check if there are any regressions, such as the "missing rotation" issue I observed in the past.
